### DISCUSSION TITLE
Coincido en que está como muy plano, pensé un poner una capa por defecto...

### DISCUSSION THREAD

**Discussion by MaCuur** [grupoioa/gis_agm_series#22] [Ideas]
Se puede poner una descripción muy escueta sobre el uso de esa herramienta. y ponerle la liga a la documentación más avanzada, que ya será una pag en web con la funcionalidad de que se descargue en .PDF

O en una página (capa) que se despligue en cuanto se acceda a la página  y que una vez que se cierre o mimimice, ya se pueda ver el mapa

ahhh y que una vez que se de click, se convierta en el algo como marcador, justo en donde señala la flecha del globo que se despliega, quitándole decimales al numde la coordenada

**Comment by ma-robles** [comment]
Suena bien esa idea de la capa inicial, no entendí muy bien lo del marcador pero puede ser una buena idea también.
También de acuerdo en quitar decimales, no lo he hecho porque es un detalle menor pero lo agrego a los pendientes, igual si alguien lo quiere ir adelantando :)

**Reply by MaCuur** [reply]
ok, si llego a eso antes de que tú lo hagas, lo hago jeje
